datacenterlight/templates/datacenterlight/includes/_navbar.html

```diff
@@ -34,6 +34,17 @@
                     <a class="on-hover-border" href="{% change_lang 'de' %}">Deutsch&nbsp;&nbsp;<i class="fa fa-globe" aria-hidden="true"></i></a> {% else %}
                     <a class="on-hover-border" href="{% change_lang 'en-us' %}">English&nbsp;&nbsp;<i class="fa fa-globe" aria-hidden="true"></i></a> {% endif %}
                 </li>
+                {% if not request.user.is_authenticated %}
+                    <li>
+                        <a href="{% url 'hosting:login' %}">{% trans "Login" %}&nbsp;&nbsp;<span class="fa fa-sign-in"></span></a>
+                    </li>
+                {% else %}
+                    <li>
+                        <a href="{% url 'hosting:dashboard' %}">{% trans "Dashboard" %}</a>
+                    </li>
+                {% endif %}
+                {% if request.path == '/en-us/datacenterlight/' or request.path == '/de/datacenterlight/' %}
```

Comment:
Do we intend to do something here? If not, it can be removed I guess.

Reply:
Yes, should be removed. 👍